### PR TITLE
Fix #3077 remove jqueryui

### LINKED ISSUE #3077: Remove jquery-ui from External Import gui
### Describe the problem

jquery-ui is being removed from codebase

### Describe the solution you'd like

The External Import gui at Administration->Coding->'External Data Loads' has a gui based on jquery-ui. Need to remove the reliance of jquery-ui from this gui. Script is at interface/code_systems/dataloads_ajax.php

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #3077 

#### Short description of what this resolves:
Remove jquery-UI.
Add Bootstrap tooltip, collapse, modal, button style.

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @stu01509 ,
Your amassing quite a few "renegade" commits on your working branch. Recommend the following to clean them out (I am assuming upstream is your repo name that is pointing to the official OpenEMR repo).
```
git checkout master
git fetch upstream
git pull upstream master
git push origin master
git checkout -b fix-#3077-remove-jqueryui_2
git cherry-pick e3aff0dd0f8e533cd0e6b8837a13403751b1d487
```
Then can make another PR with that new clean branch.

File: interface/code_systems/icd10_howto.php [hunk 1]
```diff
@@ -24,16 +24,21 @@
...
-</li>
-</ol>
-</p>
+<?php Header::setupHeader(); ?>
+
+<div class="container">
+    <p>
+        <?php echo xlt("Steps to install the ICD 10 database"); ?>:
+        <ol>
+            <li><?php echo xlt("The raw data feed release can be obtained from"); ?> <b><a href="https://www.cms.gov/Medicare/Coding/ICD10/index" target="_blank" rel="noopener"><?php echo xlt("this location"); ?></a></b>
```
Comment (im-Amitto) [inline interface/code_systems/icd10_howto.php]: You can use `font-weight-bold` here and remove `<b>`

File: interface/code_systems/rxnorm_howto.php [hunk 2]
```diff
@@ -24,19 +24,25 @@
...
-<h5 class="error_msg"><?php echo xlt("NOTE: Only the full monthly RxNorm release is currently supported"); ?></h5>
-<h5 class="error_msg"><?php echo xlt("NOTE: The import can take up to several hours"); ?></h5>
-</p>
+<?php Header::setupHeader(); ?>
+
+<div class="container">
+    <p>
+        <?php echo xlt("Steps to install the RxNorm database"); ?>:
+        <ol>
+            <li><?php echo xlt("The first step is to open an account with the Unified Medical Language System web site"); ?> <b><a href="https://utslogin.nlm.nih.gov/cas/login" target="_blank" rel="noopener"><?php echo xlt("here"); ?></a></b>
```
Comment (im-Amitto) [inline interface/code_systems/rxnorm_howto.php]: You can use `font-weight-bold` here and remove `<b>`

File: interface/code_systems/rxnorm_howto.php [hunk 3]
```diff
@@ -24,19 +24,25 @@
...
+            </li>
+            <li><?php echo xlt("Then the raw data feed release can be obtained from"); ?> <b><a href="https://www.nlm.nih.gov/research/umls/rxnorm/docs/rxnormfiles.html" target="_blank" rel="noopener"><?php echo xlt("this location"); ?></a></b>
```
Comment (im-Amitto) [inline interface/code_systems/rxnorm_howto.php]: You can use `font-weight-bold` here and remove `<b>`

File: interface/main/calendar/modules/PostCalendar/pnadmin.php [hunk 4]
```diff
@@ -700,52 +700,52 @@ function postcalendar_adminmenu($menuItem)
...
+        <li class="nav-item">
+            <a class="nav-link active" href="$cacheURL">$cacheText</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" href="$systemURL">$systemText</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link" href="$categoryURL">$categoryText</a>
+        </li>
+        EOF;
```
Comment (im-Amitto) [inline interface/main/calendar/modules/PostCalendar/pnadmin.php]: Indentation is a good idea but IMHO placing `EOF;` without indentation will make it more noticable.
@bradymiller what are your thoughts about this?

File: interface/code_systems/dataloads_ajax.php [hunk 5]
```diff
@@ -36,323 +36,253 @@
...
+                <div class="card-body">
+                    <div class="status" id="<?php echo attr($db); ?>_status">
+                    </div>
+                    <div class="left_wrpr">
+                        <div class="inst_dets">
+                            <div class="inst_hdr"><?php echo xlt("Installed Release"); ?>
+                            </div>
+                            <hr>
+                            <div id="<?php echo attr($db); ?>_install_details">
+                                <div id='<?php echo attr($db); ?>_inst_loading' style='margin: 10px; display: none;'>
```
Comment (im-Amitto) [inline interface/code_systems/dataloads_ajax.php]: 'm-2(you can try 1,3) d-none' can be used here

File: interface/code_systems/dataloads_ajax.php [hunk 6]
```diff
@@ -36,323 +36,253 @@
...
+                        </div>
+                    </div>
+                    <div class="wrpr">
+                        <div class="stg_dets">
+                            <div class="stg_hdr" id="<?php echo attr($db); ?>_stg_hdr"><?php echo xlt("Staged Releases"); ?>
+                            </div>
+                            <hr>
+                            <div id="<?php echo attr($db); ?>_stage_details">
+                            </div>
+                            <div id='<?php echo attr($db); ?>_stg_loading' style='margin: 10px; display: none;'>
```
Comment (im-Amitto) [inline interface/code_systems/dataloads_ajax.php]: 'd-none m-2'  can be used here

File: interface/main/calendar/modules/PostCalendar/pnadmin.php (hunk 4 above)
Comment (bradymiller) [reply interface/main/calendar/modules/PostCalendar/pnadmin.php]: This is actually part of a renegade commit and will be going away after branch gets cleaned up per my instructions above. Trying to indent EOF blocks here actually causes parsing error in PHP 7.2 (this is why travis is failing).

**Comment by stu01509** [comment]
Hi @bradymiller 

Thanks for your tips, I've been created another PR.
Please review it again :)

File: interface/code_systems/icd10_howto.php (hunk 1 above)
Comment (stu01509) [reply interface/code_systems/icd10_howto.php]: Thanks for your notice, I didn't think it can use font-weight-bold to replaced <b>
Good find 👍

**Pull request closed without merging**